.github/workflows/breaking_change_pr_template.md

```diff
@@ -0,0 +1,18 @@
+**Detailed Description**
```

Comment:
@Groxx I guess you have some ideas on this template. Merging it now. Feel free to update